src/lib/lib.jl

```diff
@@ -228,7 +228,7 @@ _pullback(cx::AContext, ::typeof(getproperty), x, f::Symbol) =
   _pullback(cx, literal_getproperty, x, Val(f))
 
 _pullback(cx::AContext, ::typeof(getfield), x, f::Symbol) =
-  _pullback(cx, literal_getproperty, x, Val(f))
+  _pullback(cx, literal_getproperty, x, Val(f), getfield)
```

Comment:
and to match my suggestion above
```suggestion
  _pullback(cx, literal_getfield, x, Val(f))
```